annoyed-alligators/socl_media/apps/timeline/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+# Create your models here.
+
+
+class Posts(models.Model):
+    post_content = models.TextField(verbose_name="content")
+    posted_by = models.ForeignKey(User, on_delete=models.CASCADE)
+    post_date_posted = models.DateTimeField(auto_now_add=True,
```

Comment:
The indentation here is wrong. You just have to align it with the opening brace above.